README.md

```diff
@@ -102,7 +107,7 @@ Add the stop modifier to the directive to stop event propagation.
 ## Key Code Alias
 
 The default key code map is based on US standard keyboard.
-This ability to provide their own key code alias for developers who using keyboards with different layouts. The alias name must be a **SINGLE CHARACTOR**.
+This ability to provide their own key code alias for developers who using keyboards with different layouts. The alias name must be a **single character**.
```

Comment:
I don't think this changes is necessary, capital letters can attract more attention.

Reply:
And in the same time are often interpreted as shouting which I understandably wanted to avoid. It is your call, if you want that reverted I'll do it right away. 😉

Reply:
ooops, not to shout out plz🤣
I'm not good at english language culture, if it's not very appropriate, just keep it.